### PR TITLE
samples: lwm2m_client: Portfolio Object support

### PR DESCRIPTION
**Pull request by jheiskan81**
Added Portfolio object support for lwm2m_client sample.

This is enabler for LwM2M conformance test purpose.

### CODE REVIEW

**Comment by jheiskan81** [comment]
@SeppoTakalo @rlubos @VeijoPesonen Please review.

**Review by SeppoTakalo — APPROVED** [review]

File: samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_portfolio_object.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+static char manufacturer_id[40] = "Host Develce Manufacturer #1";
+static char device_model[40] = "Host Device Model #1";
+static char software_version_id[40] = "Host Device Software Version #1";
+
+int lwm2m_init_portfolio_object(void)
+{
+
+	/* create switch1 object */
+	lwm2m_engine_create_obj_inst(LWM2M_PATH(16, 0));
+        lwm2m_engine_create_res_inst("16/0/0/0");
```
Comment (VeijoPesonen) [inline samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_portfolio_object.c:28]: Indentation.

Comment (jheiskan81) [reply samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_portfolio_object.c:28]: ok

**Comment by umapraseeda** [comment]
@SeppoTakalo , could you rebase as there are merge conflicts with the changelog?

**Comment by jheiskan81** [comment]
@VeijoPesonen I fix your comment and @umapraseeda I rebase also.

**Review by VeijoPesonen — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**